hadoop/pom.xml

```diff
@@ -81,8 +81,21 @@
             <groupId>org.apache.flink</groupId>
             <artifactId>flink-java</artifactId>
             <scope>provided</scope>
-            <version>0.9.1</version>
+            <version>1.7.1</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-core</artifactId>
+            <version>1.2.1</version>
```

Comment:
This is a very old Hadoop version back from 2013. We should use Version 2.8

Reply:
Could you try with this version instead:
```xml
<dependency>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-core</artifactId>
   <version>3.1.1</version>
</dependency>

Reply:
I imported that a while ago into the MEAD repo

Reply:
thanks @gustavonalle , I just updated the code!